package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-matomo-sdk",
-  "version": "0.4.0",
+  "version": "0.4.1",
```

Comment:
Please don't change the package version here. I'll do it when the next version is released.